import-service/src/services/s3.ts

```diff
@@ -0,0 +1,40 @@
+import AWS from 'aws-sdk';
+import { REGION, S3_UPLOADED_FOLDER } from '../constants';
+import { config } from '../config';
+
+const s3 = new AWS.S3({ region: REGION });
+
+const defaultOptions = {
+  Bucket: config.s3.bucketName
+};
+
+export function getSignedUrl(key) {
+  return s3.getSignedUrlPromise('putObject', {
+    ...defaultOptions,
+    Key: `${S3_UPLOADED_FOLDER}/${key}`,
+    Expires: 60,
+    ContentType: 'text/csv'
+  });
+}
+
+export function getReadableStream(key) {
+  return s3.getObject({
+    ...defaultOptions,
+    Key: key,
+  }).createReadStream();
+}
+
+export async function moveObject(oldObjectKey: string, newPath: string) {
+  const objectName = oldObjectKey.split('/').pop();
+  const newObjectKey = newPath ? `${newPath}/${objectName}` : objectName;
```

Comment:
Do this check has any sense? As I can understand we should not do anything if `newPath` is not provided

Reply:
that means that the file should be moved to root directory

Reply:
@Ivanko5417 understood